CONTRIBUTING.md

```diff
@@ -1,54 +1,50 @@
 # Want to contribute some code?
 
-## We’d really appreciate your support!
+We’d really appreciate your support! In order to keep things organized, please make sure you understand the following.
 
 - If the [Github issue](https://github.com/jamulussoftware/jamulus/issues) for your feature/bug fix already exists, write a message in that issue saying you want to work on it.
 - Otherwise, please [post on the GitHub Discussions](https://github.com/jamulussoftware/jamulus/discussions) and say that you are planning to do some coding and explain why. Then we can discuss the specification.
-- Please only start coding after we have agreed to a specification to avoid putting lots of work into something which may not get accepted later.
+- To avoid putting lots of work into something which may not get accepted later, please start coding after we have agreed to a specification.
 
 ## Jamulus project/source code general principles
 
-### Stability
+### 1. Stability
 
-This has been, and must continue to be, the most important requirement. It is particularly important for use cases such as the Worldjam where instability during a live performance would be unacceptable. The following two principles are designed to support this principle.
+Stability has been, and must continue to be, the most important requirement. In particular, instabilities during live performances such as WorldJam are not acceptable. The following principles are designed to support this.
 
-### [Keep it Simple and Stupid](https://en.wikipedia.org/wiki/KISS_principle) and [Do One Thing and Do It Well](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well)
+### 2. [Keep it Simple and Stupid](https://en.wikipedia.org/wiki/KISS_principle) and 3. [Do One Thing and Do It Well](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well)
 
-If a feature or function can be achieved in another way by another system or method, then that is preferable to building that feature into Jamulus.
+If a feature or function can be achieved in another way by another system or method, then that is preferable instead of building that feature into Jamulus. Consider having a look at e.g. the [JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md) API which allows you to communicate with the client and server from the outside.
```

Comment:
This is a rather quick jump from something generic to something specific.

```suggestion
If a feature or function can be achieved in another way by another system or method, then that is preferable instead of building that feature into Jamulus. Instead of implementing each and every feature as part of Jamulus, we focus on a stable core and implement interfaces for interaction with third-party components as necessary. One example is the [JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md) API which allows you to communicate with the client and server from the outside.
```